### PR TITLE
Backport trivial pr18

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

File: test/functional/p2p_sendheaders.py [hunk 1]
```diff
@@ -241,10 +241,6 @@ def run_test(self):
         # will occur outside of direct fetching
         test_node = self.nodes[0].add_p2p_connection(BaseNode(), services=0)
 
-        # Ensure verack's have been processed by our peer
-        inv_node.sync_with_ping()
-        test_node.sync_with_ping()
-
```
Comment (PastaPastaPasta) [inline test/functional/p2p_sendheaders.py:247]: drop 18496, depends on 18247

Comment (PastaPastaPasta) [reply test/functional/p2p_sendheaders.py]: done

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
Looks mostly fine. Found just one issue, see below.

File: src/qt/bitcoingui.cpp [hunk 2]
```diff
@@ -807,6 +807,10 @@ void BitcoinGUI::addWallet(WalletModel* walletModel)
 void BitcoinGUI::removeWallet(WalletModel* walletModel)
 {
     if (!walletFrame) return;
+
+    labelWalletHDStatusIcon->hide();
+    labelWalletEncryptionIcon->hide();
```
[the remark this answers is not in the document]
Comment (PastaPastaPasta) [reply src/qt/bitcoingui.cpp:812]: dropped

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**